### DISCUSSION TITLE
Nuget: CSX and MonoRepo projects

### DISCUSSION THREAD

**Discussion by dave-yotta** [renovatebot/renovate#22728] [Suggest an Idea]
### Tell us more.

Couple of things that would really help us:

1) support for nuget references in c# script `.csx` files.  They take the form `#r "nuget: PackageName, 1.0.0"` at the top of the script files.  Perhaps this is something we can raise a PR for?

2) Monorepo support. I don't mean when you're referencing a monorepo (that is supported <3) - I mean when your project IS a monorepo and thus dependencies and own versions must be updated across multiple repositories at once.  Currently we try to keep an eye out for breaking changes in nuget references and raise real cross-repo issues and PRs for them (or we miss it and our nightly starts failing).

**Comment by rarkins** [comment]
Can you create example/demonstration repos for these use cases?
